templates/common/ovirt/units/nodeip-configuration.service.yaml

```diff
@@ -0,0 +1,48 @@
+name: nodeip-configuration.service
+enabled: true
```

Comment:
Do you want to disable this service in the case of a UPI install? That is what we do for vSphere, but not sure if applicable for your case: https://github.com/openshift/machine-config-operator/blob/release-4.6/templates/common/vsphere/units/nodeip-configuration.service.yaml

Reply:
Hi thanks for the review,
I added the enabled logic but removed the ifs from the "contents" since I think that if the service is disabled it doesn't matter